File: app/src/main/java/com/wolkowiczmateusz/android/cleanmvpretrofitrxdaggerboilerplate/domain/interactors/types/UseCaseFlowable.java
package com.wolkowiczmateusz.android.cleanmvpretrofitrxdaggerboilerplate.domain.interactors.types;

import io.reactivex.Flowable;

public interface UseCaseFlowable<T, Params> {

    Flowable<T> runUseCase(Params... params);
}

File: app/src/main/java/com/wolkowiczmateusz/android/cleanmvpretrofitrxdaggerboilerplate/domain/interactors/types/UseCaseFlowable.kt
package com.wolkowiczmateusz.android.cleanmvpretrofitrxdaggerboilerplate.domain.interactors.types

import io.reactivex.Flowable

interface UseCaseFlowable<T, Params> {

    fun runUseCase(vararg params: Params): Flowable<T>
}
